slackbot/dispatcher.py

```diff
@@ -10,10 +10,14 @@
 import six
 from slackbot.manager import PluginsManager
 from slackbot.utils import WorkerPool
+from slackbot import settings
 
 logger = logging.getLogger(__name__)
 
-AT_MESSAGE_MATCHER = re.compile(r'^\<@(\w+)\>:? (.*)$')
+never_matches_anything = 'a^'
+regex_aliases = '|'.join([re.escape(s) for s in settings.ALIASES.split(',')]) if hasattr(settings, 'ALIASES') else never_matches_anything
+
+AT_MESSAGE_MATCHER = re.compile(r'^(?:\<@(\w+)\>|({})):? (.*)$'.format(regex_aliases))
```

Comment:
I'd rather do it like this:

```python
if hasattr(settings, 'ALIASES'):
   AT_MESSAGE_MATCHER = ...
else:
  AT_MESSAGE_MATCHER = ...
```

The `never_matches_anything` trick is cool, but not so good for code readability.